procurement_purchase_no_grouping/models/res_config_settings.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 - Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    """Configuration of default value for procurement purchase grouping."""
+
+    _inherit = "res.config.settings"
+    _description = "Procurement purchase grouping settings"
+
+    procured_purchase_grouping = fields.Selection(
+        related="company_id.procured_purchase_grouping",
+        string="Procured purchase grouping",
+        readonly=False,
+        help="Select the behaviour for grouping procured purchases for the "
```

Comment:
Don't add the help nor the string, as it's taken from the related field.

Reply:
OK, will do.